### PR TITLE
added percona xtrabackup restore doc

### LINKED ISSUE #1646: database backup: Procedure for disaster recovery and stored procedure
Would be great to have some documentation on how to recover from database backups in case of a disaster.
Also, DB backups should probably contains stored procedures

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
added percona xtrabackup restore doc

# Impacts
documentation generation

# Issue
fixes #1646 

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -4169,4 +4169,47 @@ Again, when executed without any arguments, a help screen is shown.
...
+
+  # xbstream -x < packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream
+
+Once done, you should have a lot of files that were extracted in the restore dir. Now, lets place the xbstream back in the backup directory
+
+  # mv packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream ../
+
+Next, install qpress (available from the percona repository) and process any qp file that were extracted:
+
+  # yum install qpress --enablerepo=percona-release-`uname -m`
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc:4179]: debian

Comment (fdurand) [reply docs/PacketFence_Administration_Guide.asciidoc:4179]: Something like that:
apt-key adv --keyserver keys.gnupg.net --recv-keys 1C4CBDCDCD2EFD2A
echo 'deb http://repo.percona.com/apt VERSION main' >> /etc/apt/sources.list
echo 'deb-src http://repo.percona.com/apt VERSION main' >> /etc/apt/sources.list
apt-get update
apt-get install qpress

**Comment by julsemaan** [comment]
fixaide

**Comment by fdurand** [comment]
Centke

**Pull request merged**